packages/Textarea/src/textarea.scss

```diff
@@ -48,6 +48,7 @@
     border: 1px solid var(--ids-textarea-border-color);
     border-radius: var(--ids-textarea-border-radius);
     color: var(--ids-textarea-color);
+    display: block;
```

Comment:
I don't understand why a `display: block` here?

Reply:
otherwise we have extra space on the bottom of the textarea. it becomes noticeable when you have an error showing.